app/src/main/java/com/example/csc207courseproject/data_access/CohereDataAccessObject.java

```diff
@@ -0,0 +1,108 @@
+package com.example.csc207courseproject.data_access;
```

Comment:
To keep our code organized, I would create a cohere package under data_access and put this DAO in that package. I would also create an Exception class which extends the DataAccessException class.